wasi.go

```diff
@@ -4,33 +4,21 @@ import (
 	"context"
 	"fmt"
 	"io"
+	"io/fs"
 
 	internalwasi "github.com/tetratelabs/wazero/internal/wasi"
 	internalwasm "github.com/tetratelabs/wazero/internal/wasm"
 	"github.com/tetratelabs/wazero/wasi"
 	"github.com/tetratelabs/wazero/wasm"
 )
 
-// WASIDirFS returns a file system (a wasi.FS) for the tree of files rooted at
-// the directory dir. It's similar to os.DirFS, except that it implements
-// wasi.FS instead of the fs.FS interface.
-func WASIDirFS(dir string) wasi.FS {
-	return internalwasi.DirFS(dir)
-}
-
-func WASIMemFS() wasi.FS {
-	return &internalwasi.MemFS{
-		Files: map[string][]byte{},
-	}
-}
-
 type WASIConfig struct {
-	stdin    io.Reader
-	stdout   io.Writer
-	stderr   io.Writer
-	args     []string
-	environ  map[string]string
-	preopens map[string]wasi.FS
```

Comment:
no reason to use a map because fs.FS can be composed by other libraries. plus we use things like path maps we end up needing to consider overlays. let fs.FS handle that